chemprop/data/scaffold.py

```diff
@@ -120,7 +120,7 @@ def scaffold_split(data: MoleculeDataset,
                      f'val scaffolds = {val_scaffold_count:,} | '
                      f'test scaffolds = {test_scaffold_count:,}')
 
-    if logger is not None:
+    if logger is not None and data[0].atom_targets is None and data[0].bond_targets is None:
```

Comment:
What's this? Why is the logger disabled for atom/bond operation?

Reply:
I think atom- and bond-level targets don't need to process log_scaffold_stats funciton.